tika-bundle/pom.xml

```diff
@@ -45,6 +45,21 @@
         </repository>
     </repositories>
 
+    <dependencyManagement>
```

Comment:
✏️ I think instead of this - which introduces a little bit of fragility here just to affect the manifest generated - it might be better to be explicit in the `ImportPackage` instruction:
```xml
org.bouncycastle.cert;version="[2.6.1,3.0)";resolution:=optional,
org.bouncycastle.cert.jcajce;version="[1.6.0,2.0)";resolution:=optional,
org.bouncycastle.cert.ocsp;version="[2.6.1,3.0)";resolution:=optional,
org.bouncycastle.cms.bc;version="[2.6.1,3.0)";resolution:=optional,
org.bouncycastle.operator;version="[2.6.1,3.0)";resolution:=optional,
org.bouncycastle.operator.bc;version="[2.6.1,3.0)";resolution:=optional,
org.bouncycastle.tsp;version="[2.6.1,3.0)";resolution:=optional,
```

Reply:
If we're embedding the tika-parsers module will this effect what that pulls in?

Reply:
Ah. I see now that `Embed-Transitive` is true. You're right that this is the right solution. Though you could explicitly exclude the bouncy castle libraries from the embed instead. That might give you the correct artifact.

Reply:
I tried excluding `bcprov-jdk15on` from the embedded dependencies and importing only these versions of bouncy castle, but Karaf threw this bundle wiring exception when it started up. 🤷‍♂️
```missing requirement [org.codice.thirdparty.tika-bundle/1.18.0.2] osgi.wiring.package; filter:="(osgi.wiring.package=bsh)" ```

Reply:
Blecch. Okay. Go with the solution that works most easily with the bundle plugin.